docs/README.md

```diff
@@ -0,0 +1,231 @@
+# Consumer documentation
```

Comment:
These docs should be housed in one of the packages, either terra-graphs or carbon-graphs. I'd recommed using terra graphs and enabling the docs to be deployed via terra-dev-site.

Reply:
this goes for all docs in the 'docs' folder.